CSS/main.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i");
+
+html {
+  background-color: #e5e5e5;
+}
+
+header {
+  flex-wrap: wrap;
+  background-color: white;
+}
+
+nav {
+  height: 48px;
+  display: flex;
+  width: auto;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.logo {
+  display: flex;
+  margin-left: 24px;
+  color: #6070ff;
+  justify-items: center;
+}
+
+.logo h2 {
+  font-family: "Poppins", "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.logo a:link,
+.logo a:visited,
+.logo a:hover,
+.logo a:active {
+  text-decoration: none;
+  color: inherit;
+}
+
+.menu {
+  margin-right: 24px;
+  display: flex;
+}
+
+.hello {
+  background-color: white;
+  border-bottom-left-radius: 50px;
+}
+
+.hellosection {
+  border-bottom-left-radius: 30px;
+  display: flex;
+  flex-direction: column;
+  height: 587px;
+  background-image: url(images/bground.svg);
+}
+
```

Comment:
- [ ] Please, consider fixing the background - it is actually not like in the Figma template

> Figma design
<img width="223" alt="Screen Shot 2022-03-20 at 00 24 57" src="https://user-images.githubusercontent.com/49064106/159137544-a2d575a7-f0ac-419a-9851-ba638b648576.png">

> Your version
<img width="316" alt="Screen Shot 2022-03-20 at 00 25 07" src="https://user-images.githubusercontent.com/49064106/159137550-c8cbb6de-9030-4d29-8775-ffe6f5c895f9.png">

Reply:
Modification done but BG doesn't appear!